src/plugins/home/public/application/components/tutorial_directory.js

```diff
@@ -190,8 +198,9 @@ class TutorialDirectoryUi extends React.Component {
   };
 
   renderTabContent = () => {
-    if (this.state.selectedTabId === SAMPLE_DATA_TAB_ID) {
-      return <SampleDataSetCards addBasePath={this.props.addBasePath} />;
+    const tab = this.tabs.find(({ id }) => id === this.state.selectedTabId);
+    if (tab?.content !== undefined) {
+      return tab.content;
```

Comment:
NIT: `if (tab?.content)`

Reply:
Updated in https://github.com/elastic/kibana/pull/100863/commits/b532118bd2a9a35510b3a5ca1389498de89f1fdd